src/components/ADempiere/Field/FieldOptions/calculator/buttons.js

```diff
@@ -1,6 +1,6 @@
 // ADempiere-Vue (Frontend) for ADempiere ERP & CRM Smart Business Solution
 // Copyright (C) 2017-Present E.R.P. Consultores y Asociados, C.A.
-// Contributor(s): Edwin Betancourt edwinBetanc0urt@hotmail.com www.erpya.com
```

Comment:
Please remove this file and just change when touch it

Reply:
changes applied